### PR TITLE
add test cases for view and view-department

### PR DESCRIPTION
**Pull request by ngshijun**

### CODE REVIEW

File: src/test/java/coydir/logic/commands/ViewCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                .withPhone("94351253").withRating("3").withTags("friends").withTotalLeave(15)
+                .withEmployeeId("1").build();
+
+        model.addPerson(personToView);
+        expectedModel.addPerson(personToView);
+        ViewCommand viewCommand = new ViewCommand(INDEX_FIRST_PERSON);
+        String expectedMessage = String.format(ViewCommand.MESSAGE_VIEW_PERSON_SUCCESS, personToView.getName());
+        assertViewingCommandSuccess(viewCommand, model, expectedMessage, expectedModel, 0);
+    }
+}
```
Comment (kevinchangjk) [inline src/test/java/coydir/logic/commands/ViewCommandTest.java:33]: Can add negative test cases also? For invalid index, invalid format, etc.

File: src/test/java/coydir/logic/commands/ViewDepartmentCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    public void viewSalesDepartmentSuccess() {
+        ViewDepartmentCommand viewDepartmentCommand = new ViewDepartmentCommand("Sales");
+        String expectedMessage = String.format(
+                ViewDepartmentCommand.MESSAGE_VIEW_DEPARTMENT_SUCCESS, "Sales");
+        assertViewDepartmentCommandSuccess(viewDepartmentCommand, model, expectedMessage,
+                expectedModel, "Sales");
+    }
+
+    @Test
+    public void viewTechnologyDepartmentSuccess() {
```
Comment (kevinchangjk) [inline src/test/java/coydir/logic/commands/ViewDepartmentCommandTest.java:160]: Woah that's a lot.

File: src/test/java/coydir/logic/commands/ViewDepartmentCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        ViewDepartmentCommand viewDepartmentCommand = new ViewDepartmentCommand("Technology");
+        String expectedMessage = String.format(
+                ViewDepartmentCommand.MESSAGE_VIEW_DEPARTMENT_SUCCESS, "Technology");
+        assertViewDepartmentCommandSuccess(viewDepartmentCommand, model, expectedMessage,
+                expectedModel, "Technology");
+    }
+}
```
Comment (kevinchangjk) [inline src/test/java/coydir/logic/commands/ViewDepartmentCommandTest.java:167]: But same here, can add negative test cases also?

**Review by kevinchangjk — CHANGES_REQUESTED** [review]

File: src/test/java/coydir/logic/commands/ViewDepartmentCommandTest.java (hunk 3 above)
Comment (Kok-je) [inline src/test/java/coydir/logic/commands/ViewDepartmentCommandTest.java:167]: I think similar to what Kevin mentioned. Perhaps can add more failure cases when the user messes up the view command, like view non-existent department or something

**Review by Kok-je — COMMENTED** [review]
Minor changes otheriwse LGTM

**Comment by ngshijun** [comment]
Also I fixed the error message for `view` command. So now it will display `invalid index` instead of `invalid id`

File: src/main/java/coydir/commons/core/Messages.java [hunk 4]
```diff
@@ -6,6 +6,8 @@
 public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = " is an unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+
```
Comment (kevinchangjk) [inline src/main/java/coydir/commons/core/Messages.java:9]: Wait why add empty line here?

**Review by kevinchangjk — APPROVED** [review]
LGTM.

**Review by Kok-je — APPROVED** [review]
LGTM

File: src/main/java/coydir/commons/core/Messages.java (hunk 4 above)
Comment (ngshijun) [reply src/main/java/coydir/commons/core/Messages.java:9]: will update in the next PR with dg update

**Pull request merged**